src/board_controller/ant_neuro/build.cmake

```diff
@@ -29,6 +29,8 @@ if (MSVC)
 endif (MSVC)
 if (UNIX AND NOT APPLE AND NOT ANDROID)
     file (COPY "${CMAKE_CURRENT_SOURCE_DIR}/third_party/ant_neuro/linux/${ANT_LIB_NAME}" DESTINATION "${CMAKE_CURRENT_SOURCE_DIR}/csharp_package/brainflow/brainflow/lib/")
+    file (COPY "${CMAKE_CURRENT_SOURCE_DIR}/third_party/ant_neuro/linux/${ANT_LIB_NAME}" DESTINATION "${CMAKE_CURRENT_SOURCE_DIR}/swift_package/BrainFlow/lib/")
+    file (COPY "${CMAKE_CURRENT_SOURCE_DIR}/third_party/ant_neuro/linux/${ANT_LIB_NAME}" DESTINATION "${CMAKE_CURRENT_SOURCE_DIR}/swift_package/BrainFlowCI/Frameworks/")
```

Comment:
we should copy only to BrainFlow/lib and from BrainFlow/lib we need to create a standalone swift binding which will be used by applications like BrainFlowCI, the same for all build.cmake files

Reply:
For the Xcode build the dylibs need to be located within the project's Frameworks directory. I tried using relative paths for the dylibs from within the build settings, but I could not get it to work.

Reply:
Brainflow dylibs must be copied, its correct. But from cmake they should be copied only to bindings. And it should be swift package or bash commands in CI pipelines or users who copy them from binding to their swift apps

Reply:
There can be a lot of apps using swift binding and build scripts for cpp part should not copy libs to all of them. From cmake we copy only to bindings and its up to binding to decide how to ensure that apps will get it. For python for example we pack them to whl, for Matlab ask users to download and unpack tar file with libs

Reply:
Got it. I will remove the extraneous copies.